bin/cmsl1t

```diff
@@ -0,0 +1,128 @@
+#!/bin/env python
+from __future__ import print_function
+import ROOT
+import os
+from datetime import datetime
+from cmsl1t.utils.timers import timerfunc_log_to
+from cmsl1t.config import ConfigParser
+import click
+from importlib import import_module
+import logging
+logger = logging.getLogger(__name__)
+logging.getLogger("rootpy.tree.chain").setLevel(logging.WARNING)
+
+TODAY = datetime.now().timetuple()
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+ROOT.gROOT.SetBatch(1)
+ROOT.TH1.SetDefaultSumw2(True)
+separator = '=' * 80
+section = [separator, '{0}', separator]
+section = '\n'.join(section)
+
+
+@timerfunc_log_to(logger.info)
+def process_tuples(config, nevents, analyzers):
+    # Open the data files
+    click.echo(section.format("Loading data"))
+
+    input_files = config.get('input', 'files')
+    click.echo("Input files:")
+
+    if len(input_files) > 10:
+        click.echo(input_files[:10], "... and",
+                   len(input_files) - 10, "more files")
+    else:
+        click.echo(input_files)
+
+    from cmsl1t.playground.eventreader import EventReader
+    reader = EventReader(input_files, events=nevents)
+
+    results = [analyzer.prepare_for_events(reader) for analyzer in analyzers]
+    check(results, analyzers, 'prepare_for_events')
+
+    click.echo(section.format("Processing events"))
+    # Fill the histograms from the tuples
+    counter_rate = nevents / 10 if nevents <= 10000 else 1000
```

Comment:
Since the default value of nevents is now `-1`  (previously it was 1000) this line is now screwy, and we end up printing every event as `<evt_no> of -1`:
```
15500 of -1
15501 of -1
15502 of -1
15503 of -1
15504 of -1
15505 of -1
```

The behaviour I wanted to implement before would require you to know how many events you will visit.  But since that would require you to open every input file, perhaps if `nevents==-1` we should set `counter_rate` to `1000` ? Or we change it's value in the loop to 1e4 once we get above 1e4?  Or we just keep it fixed as 1e3?  Up to you, but the way this is here should probably be changed.